iocore/net/UnixNetVConnection.cc

```diff
@@ -91,7 +91,7 @@ read_signal_and_update(int event, UnixNetVConnection *vc)
     case VC_EVENT_ACTIVE_TIMEOUT:
     case VC_EVENT_INACTIVITY_TIMEOUT:
       Debug("inactivity_cop", "event %d: null read.vio cont, closing vc %p", event, vc);
-      Warning("read: Closing orphaned vc %p", vc);
+      Warning("read: Closing orphaned vc vc=%p cont=%p event=%d", vc, vc->read.vio.cont, event);
```

Comment:
Is orphaned VC a new name for inactivity cop? Why are we throwing a warning for a typical inactivity cop scenario?

Reply:
Only throwing a warning in case there was no continuation associated with the VC.  Should not normally happen.

Reply:
This is the normal scenario as the state machine may decide to go away for several reasons, but iocore might still keep the VC for clearing the events. Please let me know if I am missing something.

Reply:
In my experience this is a pretty rare occurrence, but I'm ok with taking out the warning message.

Reply:
Pushed another commit to remove the warning.